### DISCUSSION TITLE
Not able to proceed with game

### DISCUSSION THREAD

**Discussion by Atharv181** [OpenZeppelin/ethernaut#563] [Support]
![image](https://user-images.githubusercontent.com/73999654/201619518-31710fc8-5a55-42a3-8f4c-65ebf79804f4.png)
Got this in the Hello Ethernaut challenge.

**Comment by xaler5** [comment]
Hey @Atharv181 have you tried `await contract.info()` ?

![image](https://user-images.githubusercontent.com/20795374/201879874-14de58e6-87c6-48ef-a869-e7448b414983.png)

`infoo` and `info0` are not functions of the contract

**Comment by Atharv181** [comment]
> Hey @Atharv181 have you tried `await contract.info()` ?

Yes that was the first command.

**Comment by xaler5** [comment]
@Atharv181 you didn't put `await` and I can't see `info` being called. only `infoo` and `info0` which are bot non existent.

**Comment by Atharv181** [comment]
![image](https://user-images.githubusercontent.com/73999654/201972257-b36e462e-468e-464b-a06c-f3c9fae27839.png)
![image](https://user-images.githubusercontent.com/73999654/201972708-2482f1b2-568d-4ee7-a6a5-49bfb89adbd3.png)

**Comment by xaler5** [comment]
@Atharv181 

In order for the call to suceeds you always need to put an `await` before, otherwise you need to define a callback to catch the result at some point.

As you might see `await contract.info()` worked. 

Regarding the first call `await contract.info1()` that failed. In which network are you trying these calls ?

**Comment by Atharv181** [comment]
Goerli network

**Comment by xaler5** [comment]
Can you do a last try just using the `await` variant ? not calling `contract.*` without `await`

**Comment by Atharv181** [comment]
Tried still not able to resolve

**Comment by Atharv181** [comment]
Are you able to do it on your PC?

**Comment by xaler5** [comment]
Yes I am @Atharv181

**Comment by Atharv181** [comment]
Can you share the commands you used

**Comment by xaler5** [comment]
I'm moving this to a github discussion so everyone can benefit from it whenever we figure out the solution :)
